models/pull.go

```diff
@@ -657,6 +657,23 @@ func (pr *PullRequest) IsSameRepo() bool {
 	return pr.BaseRepoID == pr.HeadRepoID
 }
 
+// GetPullRequestByHeadBranch returns all prs by head branch
+// Since there could be multiple prs to the same head branch, this function returns a slice of prs
+func GetPullRequestByHeadBranch(headBranch string, headRepo *Repository, status PullRequestStatus) (prs []*PullRequest, err error) {
+	prs = make([]*PullRequest, 0)
+	err = x.
+		Where("head_branch = ? AND head_repo_id = ? AND status = ?", headBranch, headRepo.ID, status).
+		Desc("id").
+		Find(prs)
+	if len(prs) == 0 {
```

Comment:
```suggestion
	if err != nil && len(prs) == 0 {
```

Reply:
why should we drop an error?

Reply:
a thing from @kolaente ... I think we can savely remove it